src/styles/blocks/header.scss

```diff
@@ -0,0 +1,110 @@
+.header {
+  
+  &__menu {
+    @include Desktop {
+      display: none;
+    }
+  }
+
+  &__top {
+    @include top-actions;
+  }
+
+  &__nav {
+    @include Desktop {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      list-style: none;
+      gap: 40px;
+      font-size: 18px;
+      letter-spacing: 5%;
+      font-weight: 500;
+      text-transform: capitalize;
+      
+      &--link {
+        color: $secondary-color;
+        text-decoration: none;
+      }
+    }
+    display: none;
+  }
+
+  &__main {
+    margin-bottom: 0;
+    
+    &--info {
+      @include Desktop {
+        grid-row: 1 / 2;
+      }
+    }
+
+    &--title {
+      padding-bottom: 10px;
+
+      @extend %text-h4;
+      
+      @include Tablet {
+        font-size: 36px;
+      }
+
+      @include Desktop {
+        text-align: start;
+        font-size: 60px;
+        line-height: 130%;
+        padding-bottom: 30px;
+      }
+    }
+
+    &--paragraph {
+      text-align: center;
+      padding-bottom: 30px;
+
+      @include Tablet {
+        padding-bottom: 50px;
+      }
+
+      @include Desktop {
+        text-align: left;
+      }
+    }
+
+    &--button {
+      margin: 0 auto;
+
+      @include Desktop {
```

Comment:
It's a good practice to put mixins at the end of the selector or properties block.